src/GeometryInstancing/Shaders/Vertex.glsl

```diff
@@ -0,0 +1,30 @@
+#version 330 core
+
+struct Camera_ProjView
+{
+    mat4 View;
+    mat4 Proj;
+};
+
+layout(std140) uniform projView
+{
+    Camera_ProjView field_projView;
```

Comment:
Uniforms don't need to just be a single struct, the binding for "projView" should cover whatever is defined inside of the block. So you could remove the structure above if you wanted.

Reply:
Nice. I wasnt sure about that. I a way I didnt mind because I "forced" me to use structs. Which is not necessarily a bad thing.